lib/puppetserver/ca/action/import.rb

```diff
@@ -58,47 +75,36 @@ def run(input)
           FileSystem.write_file(puppet.settings[:cacrl], loader.crls, 0640)
 
           # Puppet's internal CA expects these file to exist.
-          FileSystem.ensure_file(puppet.settings[:serial], "001", 0640)
+          FileSystem.ensure_file(puppet.settings[:serial], "0x0001", 0640)
```

Comment:
what's this  "0x0001" about?

Reply:
The serials are supposed to be in hex, to maintain what puppet is doing now.